lti_consumer/utils.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+"""
+Make '_' a no-op so we can scrape strings
```

Comment:
@mattdrayer @douglashall let me explain this. 
1- The purpose of the `_(text)` method in utils.py is only to mark static strings for translations, like XBlock fields help-text where we cannot refer to `self`.
2- `_(text)` method is helpful for us to create strings for .po files (When we use [poedit](https://poedit.net/) ) then it will look for `_` and generate a .PO. 

Lastly, runtime is service is using the xBlock `self`and we are not able to get the reference of `self` for Xblock fields texts. 

Let me know If I am missing something.